### PR TITLE
Add flag --installed to command modules

### PR DESCRIPTION
**Pull request by Cortey**
**Description**

Changes proposed in this pull request:

- add flag `--installed` which shows only the modules that are installed  to the command `modules`

**Related issue(s)**

#2117

### CODE REVIEW

File: internal/cmd/modules/modules.go [hunk 1]
```diff
@@ -135,6 +151,43 @@ func listManagedModules(cfg *modulesConfig) ([]string, clierror.Error) {
...
+	template, respErr := handleResponse(err, resp, template)
+	if respErr != nil {
+		return nil, clierror.WrapE(respErr, clierror.New("while handling response"))
+	}
+
+	var out []string
+	for _, rec := range template {
+		short := strings.Split(rec.Versions[0].ManagerPath, "/")
+		version := rec.Versions[0].Version
+		deployment, err := cfg.KubeClient.Static().AppsV1().Deployments("kyma-system").Get(cfg.Ctx, short[len(short)-1], metav1.GetOptions{})
```
Comment (anoipm) [inline internal/cmd/modules/modules.go:172]: What is `short[len(short)-1]`? It's hard to understand. Could we name this object (i.e. `managerName` or something similar)?

File: internal/cmd/modules/modules.go [hunk 2]
```diff
@@ -135,6 +151,43 @@ func listManagedModules(cfg *modulesConfig) ([]string, clierror.Error) {
...
+		if err != nil && !errors.IsNotFound(err) {
+			msg := "while getting the " + short[len(short)-1] + " deployment"
+			return nil, clierror.Wrap(err, clierror.New(msg))
+		}
+		if !errors.IsNotFound(err) {
+			depVersion := strings.Split(deployment.Spec.Template.Spec.Containers[0].Image, "/")
```
Comment (anoipm) [inline internal/cmd/modules/modules.go:178]: IMO `depVersion` contains image name and tag, not deployment version. Could you find better name?

File: internal/cmd/modules/modules.go [hunk 3]
```diff
@@ -135,6 +151,43 @@ func listManagedModules(cfg *modulesConfig) ([]string, clierror.Error) {
...
+	var template model.Module
+
+	template, respErr := handleResponse(err, resp, template)
+	if respErr != nil {
+		return nil, clierror.WrapE(respErr, clierror.New("while handling response"))
+	}
+
+	var out []string
+	for _, rec := range template {
+		short := strings.Split(rec.Versions[0].ManagerPath, "/")
```
Comment (anoipm) [inline internal/cmd/modules/modules.go:170]: What is `short`? Could you find better name?

File: internal/cmd/modules/modules.go [hunk 4]
```diff
@@ -135,6 +151,43 @@ func listManagedModules(cfg *modulesConfig) ([]string, clierror.Error) {
 	return moduleNames, nil
 }
 
+func listInstalledModules(cfg *modulesConfig) ([]string, clierror.Error) {
+	resp, err := http.Get("https://raw.githubusercontent.com/kyma-project/community-modules/main/model.json")
```
[the remark this answers is not in the document]
Comment (MichalKalke) [reply internal/cmd/modules/modules.go]: It is extracted now.

File: internal/cmd/modules/modules.go [hunk 5]
```diff
@@ -57,6 +59,7 @@ func NewModulesCMD(kymaConfig *cmdcommon.KymaConfig) *cobra.Command {
 
 func runModules(cfg *modulesConfig) clierror.Error {
```
[the remark this answers is not in the document]
Comment (MichalKalke) [reply internal/cmd/modules/modules.go:62]: We will do it in the next PR where we will do a refactor of whole file.

File: internal/cmd/modules/modules.go (hunk 1 above)
Comment (MichalKalke) [reply internal/cmd/modules/modules.go]: Applied suggested name.

File: internal/cmd/modules/modules.go (hunk 2 above)
Comment (MichalKalke) [reply internal/cmd/modules/modules.go]: Changed to `deploymentImage`

File: internal/cmd/modules/modules.go (hunk 3 above)
Comment (MichalKalke) [reply internal/cmd/modules/modules.go]: We changed it to `deploymentImage`

**Review by anoipm — APPROVED** [review]

**Pull request merged**